File: Project Euler/p9.py
# Special Pythagorian triplet
def main():
    s = 1000
    for a in range(s//3):
        for b in range(a, s//2):
            c = s - a - b
            if a**2 + b**2 == c**2:
                return a*b*c

if __name__ == "__main__":
    print(main())

File: Project Euler/p9.cpp
// Special Pythagorian triplet
#include <bits/stdc++.h>

using namespace std;

int main() {
    int s = 1000;
    for (int a = 1; a < s/3; a++) {
        for (int b = a; b < s/2; b++) {
            int c = s - a - b;
            if (pow(a,2) + pow(b,2) == pow(c, 2)) {
                cout << a*b*c << endl;
            }
        }
    }
    return 0;
}
